desktop/cmp/dash/DashContainerModel.js

```diff
@@ -74,7 +74,10 @@ export class DashContainerModel {
     @observable.ref goldenLayout;
     /** @member {DashViewModel[]} */
     @managed @observable.ref viewModels = [];
-
+    /** @member {boolean} */
```

Comment:
This is really awesome.  My only nits to consider are the following:

+Should these be @bindable?  Otherwise there is no way to change them, and no reason for them be observable

+ Should we have a separate property to govern the rename from 'contentLocked' (e.g. 'renameLocked', for symmetry).  Seems like we are lumping that together with `contentLocked` when apps might want to allow it seperately.